pkg/collector/corechecks/oracle/activity.go

```diff
@@ -189,6 +187,13 @@ func (c *Check) SampleSession() error {
 	err := selectWrapper(c, &sessionSamples, activityQuery)
 
 	if err != nil {
+		if strings.Contains(fmt.Sprintf("%s", err), "character string buffer too small") {
```

Comment:
The error text can in the local language. We have to parse the error code instead.

Reply:
`err.Error()` stringifies the erorr message. It's more readible than `fmt.Sprintf`.